eox_core/cms/views.py

```diff
@@ -29,32 +34,40 @@ def management_view(request):
     """
     Renders the course management view.
     """
-    if 'text/html' in request.META.get('HTTP_ACCEPT', '') and request.method == 'GET':
+    if 'text/html' in request.META.get('HTTP_ACCEPT', ''):
         org_list = get_all_orgs()
+        details_fields = get_course_details_fields()
+        course_key = get_first_course_key()
+        request_timeout_value = getattr(settings, 'EOX_CORE_COURSE_MANAGEMENT_REQUEST_TIMEOUT', 500)
```

Comment:
Since you are already setting a default on the `common.py` module I think this default is not necessary. Also if it was, you are setting a different value. The best option is to define it only once (in the common.py).